extensions/google-translate/src/actions.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { Action, ActionPanel, getPreferenceValues } from "@raycast/api";
+
+enum DefaultActionPreference {
+  CopyToClipboard = "copy",
+  PasteToApp = "paste",
+}
+
+interface Preferences {
+  defaultAction?: DefaultActionPreference;
+}
```

Comment:
Could we rely on `ExtensionPreferences` type which is auto generated instead? Check `raycast-env.d.ts`

Change line 4 in `tsconfig.json` to `"include": ["src/**/*", "raycast-env.d.ts"],`

Reply:
Done